cmd/lncli/walletrpc_active.go

```diff
@@ -396,3 +402,329 @@ func labelTransaction(ctx *cli.Context) error {
 
 	return nil
 }
+
+// fundPsbtResponse is a struct that contains JSOn annotations for nice result
+// serialization.
+type fundPsbtResponse struct {
+	Psbt  string                 `json:"psbt"`
+	Locks []*walletrpc.UtxoLease `json:"locks"`
+}
+
+var fundPsbtCommand = cli.Command{
+	Name:     "fundpsbt",
```

Comment:
nit: another option is to name the commands `lncli wallet psbt fund` and `lncli wallet psbt finalize` to enforce better grouping. idk how much nesting our cli lib allows tho...

Reply:
Good idea, done.